### PR TITLE
global: user_id in holdings

### PR DESCRIPTION
**Pull request by mvesper**
- Adds user_id to Holding object. Defaults to flask_logn.current_user.

### CODE REVIEW

File: tests/test_holdings.py [hunk 1]
```diff
@@ -60,3 +63,25 @@ def test_holdings(app, db):
...
+    # Loan an item
+    item = Item.create({})
+
+    assert not item.holdings
+
+    func(item, schema().dump({}).data)
+    item.commit()
+
+    assert len(item.holdings) == 1
+    assert 'start_date' in item.holdings[0]
```
Comment (jirikuncar) [inline tests/test_holdings.py]: `holding = item.holdings[0]` is more green 🌳

File: tests/test_holdings.py [hunk 2]
```diff
@@ -60,3 +63,25 @@ def test_holdings(app, db):
     assert len(item.holdings) == 1
     assert id2 not in item.holdings
     assert id1 in item.holdings
+
+
+@pytest.mark.parametrize(('func', 'schema'), [
```
Comment (jirikuncar) [inline tests/test_holdings.py]: Is `func` an `action`?

Comment (mvesper) [inline tests/test_holdings.py]: `func` is basically either `loan_item` or `request_item`

Comment (jirikuncar) [inline tests/test_holdings.py]: so we can call it `action`, can't we?

**Pull request merged**